qemu/tests/cfg/blockdev_mirror_cancel_ready_job_with_io.cfg

```
@@ -0,0 +1,45 @@
+# Storage backends:
+#   filesystem, iscsi_direct, ceph, nbd, gluster_direct
+# The following testing scenario is covered:
+#   blockdev-mirror speed test
+#     The mirror image is a local image(filesystem)
+
+- blockdev_mirror_cancel_ready_job_with_io:
+    only Linux
+    only filesystem iscsi_direct ceph nbd gluster_direct
+    start_vm = no
+    qemu_force_use_drive_expression = no
+    type = blockdev_mirror_cancel_ready_job_with_io
+    virt_test_type = qemu
+    images += " data1"
+    source_images = data1
+    target_images = mirror1
+    remove_image_data1 = yes
+    force_create_image_data1 = yes
+    backup_options_data1 = sync
+    sync = full
+    storage_pools = default
+    storage_pool = default
+    write_file_cmd = 'cd /tmp && nohup dd if=/dev/urandom of=file bs=1K count=512000 &'
```

```suggestion
    write_file_cmd = 'cd /tmp && dd if=/dev/urandom of=file bs=1K count=512000'
```
And delete the line 22 ~ 23:
```
       finally:
            session.close()
```
so we could not use `nohup` and `&` since using `sendline` in command for background.
But your approach is also good to me.

---

I prefer your way to use sendline, for it's shorter, but I found that tc failed to detect dd running in 30s